src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -165,8 +166,44 @@ public Object instanceMainPostParseArgs() {
 
         // set the temp directory as a java property, checking for existence and read/write access
         final Path p = tmpDir.toPath();
+        Path tempFilePath = null;
         try {
             p.getFileSystem().provider().checkAccess(p, AccessMode.READ, AccessMode.WRITE);
+
+            // Warn if there's anything that prevents execution in the tmp dir because some tools need that
+            // This test relies on the file system supporting posix file permissions
+            if(p.getFileSystem().supportedFileAttributeViews().contains("posix")) {
```

Comment:
What should be the course of action if that happens?  Catch it and move on?

Reply:
Well, we won't even get to here in that case, since we've already exploded on the check above due to not supporting write access.  Just a weird edge case.

Reply:
I think catch and continue was the right choice.